homeassistant/components/roomba/translations/de.json

```diff
@@ -31,10 +31,9 @@
             },
             "manual": {
                 "data": {
```

Comment:
No need to change anything but strings.json and `en.json` as the rest will be overwritten anyways by lokalize

https://developers.home-assistant.io/docs/translations/

Reply:
I have reverted these changes. Thanks for the tip.